trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -61,192 +63,198 @@ describe(page, () => {
     wrapper.expectElement(tabSelector).toHaveAttribute('href', href);
   });
 
-  it('should render the premium payments tab with headings, text, table and buttons', () => {
-    const wrapper = getWrapper();
-    const premiumPaymentsTabSelector = 'div#premium-payments';
-
-    wrapper.expectElement(premiumPaymentsTabSelector).toExist();
-
-    wrapper.expectText(`${premiumPaymentsTabSelector} h2[data-cy="premium-payments-heading"]`).toRead('Premium payments');
-    wrapper
-      .expectText(`${premiumPaymentsTabSelector} p`)
-      .toMatch(/Enter received payments against reported fees by selecting them and then selecting the 'Add a payment' button./);
-    wrapper
-      .expectText(`${premiumPaymentsTabSelector} p`)
-      .toMatch(
-        /When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when you select the 'Generate keying data' button./,
-      );
-
-    wrapper.expectElement(`${premiumPaymentsTabSelector} form[data-cy="premium-payments-form"]`).toExist();
-
-    wrapper.expectElement(`${premiumPaymentsTabSelector} div.govuk-button-group`).toExist();
-
-    wrapper.expectElement(`${premiumPaymentsTabSelector} input[data-cy="add-a-payment-button"]`).toExist();
-    wrapper.expectElement(`${premiumPaymentsTabSelector} input[data-cy="add-a-payment-button"]`).toHaveAttribute('value', 'Add a payment');
-    wrapper
-      .expectElement(`${premiumPaymentsTabSelector} input[data-cy="add-a-payment-button"]`)
-      .toHaveAttribute('formaction', `/utilisation-reports/${reportId}/add-payment`);
-
-    wrapper.expectElement(`${premiumPaymentsTabSelector} input[data-cy="generate-keying-data-button"]`).toExist();
-    wrapper.expectElement(`${premiumPaymentsTabSelector} input[data-cy="generate-keying-data-button"]`).toHaveAttribute('value', 'Generate keying data');
-    wrapper.expectElement(`${premiumPaymentsTabSelector} input[data-cy="generate-keying-data-button"]`).hasClass('govuk-button--secondary');
-    wrapper
-      .expectElement(`${premiumPaymentsTabSelector} input[data-cy="generate-keying-data-button"]`)
-      .toHaveAttribute('formaction', `/utilisation-reports/${reportId}/check-keying-data`);
-
-    wrapper.expectElement(`${premiumPaymentsTabSelector} table[data-cy="premium-payments-table"]`).toExist();
-  });
-
-  it('should render the facility ID filter input', () => {
-    const wrapper = getWrapper();
-    wrapper.expectElement('[data-cy="facility-filter-input"]').toExist();
-    wrapper.expectText('[data-cy="facility-filter-form"]').toContain('Filter by facility ID');
-  });
+  describe('premium payments tab', () => {
+    it('should render the premium payments tab with headings, text, table and buttons', () => {
```

Comment:
it would be nice to have a different it for each element being tested, eg it for header, it for text etc, but to keep it consistent is fine to leave it as is for now

Reply:
Agree we should test these in smaller groups - leaving as is for now as this change is just to nest these in a describe block for readability